index.html

```diff
@@ -4,11 +4,179 @@
+        <li class="top-header-item">
+            <a class="top-header-item-link" href="http://github.com/raihan2bd"><img class="top-social-icon" src="./images/github.png" alt="Github"/></a>
+        </li>
+        <li class="top-header-item">
+          <a class="top-header-item-link" href="http://linkedin.com/in/raihan2bd"><img class="top-social-icon" src="./images/linkedin.png" alt="Linkedin"/></a>
+        </li>
+        <li class="top-header-item">
+          <a class="top-header-item-link" href="#">English</a>
+        </li>
+        <li class="top-header-item">
+          <a class="top-header-item-link" href="#">Mypage</a>
+        </li>
+        <li class="top-header-item">
+          <a class="top-header-item-link" href="#">Logout</a>
+        </li>
+      </ul>
+    </div>
+    <nav class="navbar">
+      <div class="menu-icon"><img src="./images/menu-icon.png" alt="menu" /></div>
+      <a class="brand" href="index.html"><img src="./images/logo.png" alt="Code With Raihan" /></a>
+      <!-- Navigation Item for mobile -->
+      <ul class="nav-item-group m-nav">
+        <span class="cross-icon"><img src="./images/cross-icon.png" alt="x" /></span>
+        <li class="nav-item">
+          <a href="about.html" class="nav-link">About</a>
+        </li>
+        <li class="nav-item">
+          <a href="index.html#program" class="nav-link">Program</a>
+        </li>
+        <li class="nav-item">
+          <a href="index.html#tutor" class="nav-link">Tutors</a>
+        </li>
+        <li class="nav-item">
+          <a href="index.html#sponser" class="nav-link">Sponser</a>
+        </li>
+        <li class="nav-item">
+          <a href="index.html#hero" class="nav-link nav-box">Join Program</a>
+        </li>
+      </ul>
+      <!-- Navigation Items for Desktop -->
+      <ul class="nav-item-group d-nav">
+        <li class="nav-item">
+          <a href="about.html" class="nav-link">About</a>
+        </li>
+        <li class="nav-item">
+          <a href="index.html#program" class="nav-link">Program</a>
+        </li>
+        <li class="nav-item">
+          <a href="index.html#tutor" class="nav-link">Tutors</a>
+        </li>
+        <li class="nav-item">
+          <a href="index.html#sponser" class="nav-link">Sponser</a>
+        </li>
+        <li class="nav-item">
+          <a href="index.html#hero" class="nav-link nav-box">Join Program</a>
+        </li>
+      </ul>
+  </header> <!-- Heder end here -->
+
+  <!-- Main container start Form here -->
+  <main class="container">
+    <!-- Hero Section -->
+    <section id="hero" class="hero">
+      <h3 class="primary-color hero-greeting">"Hello Tech Lovers"</h3>
+      <h1 class="hero-title">BECOME A PROFESSIONAL<br> FULLSTACK DEVELOPER</h1>
+      <p class="hero-details">Boost your career joining our highly professional fullstack Bootcamp and start a remote international fulltime job. Through out this Bootcamp your learning path will be practical and beginner friendly.</p>
```

Comment:
**_kindly fix these design issues in your index.html header to match the design🙏  :_**

- [ ] Kindly Add some space to the left for the whole content
- [ ] Kindly Increase The Size of the small title on top of the main one
- [ ] kindly update the header description style including the size and the background color, border etc ...  to match the design 
- [ ] kindly remove the button in your header and replace it with some information like shown in the design

your design : 
![image](https://user-images.githubusercontent.com/89282221/203381669-b14bd144-ec3e-4a17-a06a-23b7d16ee8f6.png)


required design : 
![image](https://user-images.githubusercontent.com/89282221/203382013-ec4122b1-3411-4a6d-93c1-e875adab95b1.png)